WordPress/src/main/res/values/strings.xml

```diff
@@ -116,6 +116,7 @@
     <string name="status_and_visibility">Status &amp; Visibility</string>
 
     <string name="button_not_now">Not now</string>
+    <string name="button_skip">Skip</string>
```

Comment:
We already have this: https://github.com/wordpress-mobile/WordPress-Android/blob/cec021104d2155042d51a6660e1b7ff71bd7834e/WordPress/src/main/res/values/strings.xml#L3281, perhaps we can either reuse it, or use `button_skip` there instead? Wdyt?

Reply:
Indeed, when adding this I had the plan to raise it for discussion with a comment 😊, but somehow forgot it afterwards. Thank you, I really appreciate your keen observation skill, it's awesome!

I'll use `button_skip` for home page picker. I don't foresee that we're going to make the text different for any of the two steps.

Reply:
Done ✅